app/src/main/res/layout/main_login.xml

```diff
@@ -1,14 +1,55 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent">
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Please rename this file to a readable format. Every time a layout file should be named as either "activity_xyz", "fragment_xyz", "item_xyz"